src/core/services/theming/theming.js

```diff
@@ -282,6 +357,8 @@ function ThemingProvider($mdColorPalette) {
       alwaysWatchTheme = alwaysWatch;
     },
 
+    enableBrowserColor: enableBrowserColor,
+
```

Comment:
Normally, I wouldn't fight you on this and I'd say "ok, cool" and move on...but...

I feel like we're being needlessly restrictive by not providing it. Part of the convenience of this service/provider is hiding all of the logic about which meta tags to set for which browsers.

Additionally, I don't really see a downside of exposing it. If they want to use the theme colors, then they should definitely use the `enableBrowserColor()` approach, but if they want to set it to something that isn't part of the theme (or maybe any theme), the giving them a way to set it directly would be nice.

What downsides do you see?